### PR TITLE
Samsung.AllShare version 2.1.0.12031_10

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98697)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

The installation isn't unattended:
image

Is there an installer switch to bypass this and have it install automatically?

**Comment by upintheairsheep** [comment]
> Hi @upintheairsheep,
> 
> The installation isn't unattended: image
> 
> Is there an installer switch to bypass this and have it install automatically?

Seems to be a standard installshield installer: https://docs.revenera.com/installshield26helplib/helplibrary/InstallShieldSilent.htm#:~:text=InstallShield%20Silent%20allows%20automated%20electronic,without%20any%20end%2Duser%20intervention.

File: manifests/s/Samsung/AllShare/2.1.0.12031_10/Samsung.AllShare.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: Samsung.AllShare
+PackageVersion: 2.1.0.12031_10
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=UNI_UK&OriginYN=N&ModelType=N&ModelName=UE55ES8000U&CttFileID=3809756&CDCttType=SW&VPath=SW%2F201204%2F20120402145339253%2FAllShare_2.1.0.12031_10.exe
+  Architecture: neutral
+  InstallerType: exe
+  InstallerSha256: 1B3642D5B07BDC9E3A10985BAA9507147DDC3F0C247BE27F744DECEB57E37190
+  InstallerSwitches:
+    Silent: /quiet
```
Comment (stephengillie) [inline manifests/s/Samsung/AllShare/2.1.0.12031_10/Samsung.AllShare.installer.yaml:12]:
```suggestion
    Silent: /s
```

File: manifests/s/Samsung/AllShare/2.1.0.12031_10/Samsung.AllShare.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Samsung.AllShare
+PackageVersion: 2.1.0.12031_10
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=UNI_UK&OriginYN=N&ModelType=N&ModelName=UE55ES8000U&CttFileID=3809756&CDCttType=SW&VPath=SW%2F201204%2F20120402145339253%2FAllShare_2.1.0.12031_10.exe
+  Architecture: neutral
+  InstallerType: exe
+  InstallerSha256: 1B3642D5B07BDC9E3A10985BAA9507147DDC3F0C247BE27F744DECEB57E37190
+  InstallerSwitches:
+    Silent: /s
+    SilentWithProgress: /passive
```
Comment (stephengillie) [inline manifests/s/Samsung/AllShare/2.1.0.12031_10/Samsung.AllShare.installer.yaml:13]:
```suggestion
    Silent: /S /v /qn
```

**Comment by stephengillie** [comment]
The installer responds to interrogation: 
image

**Comment by upintheairsheep** [comment]
When is it merging time?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**